src/main/java/de/tum/in/www1/artemis/web/rest/FileUploadExerciseResource.java

```diff
@@ -110,6 +119,42 @@ public ResponseEntity<FileUploadExercise> createFileUploadExercise(@RequestBody
         return ResponseEntity.created(new URI("/api/file-upload-exercises/" + result.getId())).body(result);
     }
 
+    /**
+     * POST file-upload--exercises/import: Imports an existing file upload exercise into an existing course
+     * <p>
+     * This will import the whole exercise except for the participations and Dates.
```

Comment:
Is this correct? This does not use the dates of the first exercise, but I can directly set them in the client during the import.
```suggestion
     * This will import the whole exercise except for the participations and dates.
```